SmartDeviceLink/SDLLifecycleConfiguration.m

```diff
@@ -62,6 +62,7 @@ - (instancetype)initDefaultConfigurationWithAppName:(NSString *)appName fullAppI
     _appName = appName;
     _appType = SDLAppHMITypeDefault;
     _language = SDLLanguageEnUs;
+    _hmiLanguage = SDLLanguageEnUs;
```

Comment:
These changes were not agreed upon, please revert

Reply:
OK. This has been reverted.